### PR TITLE
Allowing usage of Kubernetes ingress in existing owncloud chart

### PR DESCRIPTION
**Pull request by carmat88**
With this PR we would like to add the option of using the Kubernetes' ingresses within the already existing owncloud chart. This is useful when load balancers are not available but an ingress controller is running in a Kubernetes cluster.

### CODE REVIEW

**Comment by carmat88** [comment]
The CI errors seem to be unrelated to what we have modified:

```
Linting the Chart.yaml and values.yaml files at stable/owncloud
stable/owncloud/values.yaml
  56:3      error    wrong indentation: expected 4 but found 2  (indentation)
  57:5      error    wrong indentation: expected 6 but found 4  (indentation)
  78:3      error    wrong indentation: expected 4 but found 2  (indentation)
  88:5      error    wrong indentation: expected 6 but found 4  (indentation)
  99:5      error    wrong indentation: expected 6 but found 4  (indentation)
  106:3     error    wrong indentation: expected 4 but found 2  (indentation)
  107:5     error    wrong indentation: expected 6 but found 4  (indentation)
```

Please let me know if I am wrong and whether I could correct the above lines anyway. Thanks!

File: stable/owncloud/values.yaml [hunk 1]
```diff
@@ -8,6 +8,13 @@ image: bitnami/owncloud:10.0.3-r0
 ##
 imagePullPolicy: IfNotPresent
 
+## Allowing use of ingress controllers
+## ref: https://kubernetes.io/docs/concepts/services-networking/ingress/
+##
+ingressController:
```
Comment (unguiculus) [inline stable/owncloud/values.yaml]: Should be `ingress` instead of `ingressController`.

**Review by unguiculus — CHANGES_REQUESTED** [review]
Thanks for the PR. Please bump chart version and fix linting issues.

**Comment by mcapuccini** [comment]
@carmat88 @unguiculus I fixed requested edits, versioning, and linting problems. It should be ready to go now!

**Comment by unguiculus** [comment]
Please indent with two spaces.

**Comment by mcapuccini** [comment]
@unguiculus maybe I am missing something, but when indenting with 2 spaces the CI complains:

```
  106:3     error    wrong indentation: expected 4 but found 2  (indentation)
```

**Review by unguiculus — APPROVED** [review]

**Comment by mcapuccini** [comment]
Thanks for fixing the last things!

**Pull request merged**